torch/autograd/forward_ad.py

```diff
@@ -67,20 +68,32 @@ def make_dual(tensor, tangent, *, level=None):
 
     return torch._VF._make_dual(tensor, tangent, level=level)
 
+UnpackedDualTensor = namedtuple('UnpackedDualTensor', ['primal', 'tangent'])
+
 def unpack_dual(tensor, *, level=None):
-    r"""Function that unpacks a "dual object" to recover two plain tensors, one representing
-    the primal and the other the tangent (both are views of :attr:`tensor`. Neither of these
-    tensors can be dual tensor of level :attr:`level`.
+    r"""Unpacks a "dual object" to return a namedtuple ``(primal, tangent)`` where
+    ``primal`` is a view of :attr:`tensor`'s primal and ``tangent`` is
+    :attr:`tensor`'s tangent. Neither of these tensors can be dual tensor of level
+    :attr:`level`.
 
     This function is backward differentiable.
+
+    Example::
+
+        >>> with dual_level():
+        ...   inp = make_dual(x, x_t)
+        ...   out = f(inp)
+        ...   y, jvp = unpack_dual(out)
```

Comment:
nit do you want an example that gets just one or the other as well with `jvp = unpack_dual(out).tangent` ?